vpr/src/place/timing_place_lookup.cpp

```diff
@@ -613,6 +597,17 @@ static void fix_uninitialized_coordinates(vtr::Matrix<float>& delta_delays) {
     }
 }
 
+static void fill_impossible_coordinates(vtr::Matrix<float>& delta_delays) {
+    //Set any empty delta's to the average of it's neighbours
+    for (size_t delta_x = 0; delta_x < delta_delays.dim_size(0); ++delta_x) {
+        for (size_t delta_y = 0; delta_y < delta_delays.dim_size(1); ++delta_y) {
+            if (delta_delays[delta_x][delta_y] == IMPOSSIBLE_DELTA) {
+                delta_delays[delta_x][delta_y] = find_neightboring_average(delta_delays, delta_x, delta_y, 5);
```

Comment:
This is needed to have no `IMPOSSIBLE_DELTAs` (by using max distance as 5 in this case), right?

Reply:
Correct.  They ideal case is to have no holes in the delta_delays matrix, but it also doesn't make sense to take an average that too wide.